templates/components/multiagent/typescript/workflow/stream.ts

```diff
@@ -46,15 +45,16 @@ export async function createStreamFromWorkflowContext<Input, Output, Context>(
         closeStreams(controller);
         return;
       }
-      const text = trimStartOfStream(chunk.delta ?? "");
-      if (text) {
-        controller.enqueue(encoder.encode(text));
+      if (chunk.delta) {
+        controller.enqueue(encoder.encode(chunk.delta));
```

Comment:
can't you directly send `EngineResponse` objects here? then you don't need to do `streamToAsyncIterable`